odin/src/main/scala/org/clulab/odin/impl/OdinConfig.scala

```diff
@@ -16,10 +16,10 @@ case class OdinConfig(
 
 object OdinConfig {
 
-  val DEFAULT_GRAPH = GraphMap.STANFORD_COLLAPSED
+  val DEFAULT_GRAPH = GraphMap.UNIVERSAL_ENHANCED
 
   def empty: OdinConfig = OdinConfig()
 
   // add newly registered graph types here
-  var VALID_GRAPHS = Set[String](GraphMap.STANFORD_COLLAPSED, GraphMap.STANFORD_BASIC, GraphMap.SEMANTIC_ROLES)
+  var VALID_GRAPHS = Set[String](GraphMap.UNIVERSAL_ENHANCED, GraphMap.UNIVERSAL_BASIC, GraphMap.SEMANTIC_ROLES)
```

Comment:
Please do not remove `GraphMap.STANFORD_COLLAPSED, GraphMap.STANFORD_BASIC` (they need to be added back to `GraphMap` as well).  Those are needed to apply rules to the old formats.